controllers/toolchainstatus/toolchainstatus_controller.go

```diff
@@ -351,7 +351,11 @@ func (r *Reconciler) membersHandleStatus(logger logr.Logger, toolchainStatus *to
 			logger.Error(fmt.Errorf("member cluster %s not ready", memberCluster.Name), "the memberstatus ready condition is not true")
 			ready = false
 		}
-		r.Log.Info("adding member status", "member_name", memberCluster.Name, string(memberStatus.Conditions[0].Type), memberStatus.Conditions[0].Status)
+		if readyCond, found := condition.FindConditionByType(memberStatusObj.Status.Conditions, toolchainv1alpha1.ConditionReady); found {
```

Comment:
wouldn't it be possible to merge this new if/else statement with the one above on line 349?

Reply:
yeah, technically yes

Reply:
I'm not sure if the code is better in this way: c459cca